Core/clim-core/menu.lisp

```diff
@@ -354,7 +313,7 @@ account, and create a list of menu buttons."
   (declare (ignore contents))
   (setf (slot-value pane 'items) (copy-list (sheet-children pane)))
   (loop for child in (menu-children pane)
-	do (setf (gadget-client child) pane)))
+        do (setf (gadget-client child) pane)))
```

Comment:
since we never create classes with empty client, this loop is not necessary anymore. In a matter of fact I'd get rid of the after method in favor of

```
(defun make-menu-bar (command-table
                      &key width height
                           (max-width +fill+) max-height
                           min-width min-height)
  (let* ((menu-bar (make-pane-1 *pane-realizer* *application-frame* 'menu-bar
                               :background *3d-normal-color*
                               :width width :height height
                               :max-width max-width :max-height max-height
                               :min-width min-width :min-height min-height))
         (contents (%make-menu-contents command-table menu-bar)))
    (setf (%pane-contents menu-bar) contents
          (slot-value menu-bar 'items) (copy-list (sheet-children menu-bar)))
    menu-bar))
```

also, since this PR does cleanup, please remove menu-button-hrack-pane and make menu-bar inherit from hrack-pane directly. class I'm suggesting to remove is not referenced anywhere else in the codebase.